stats/test/platform-routes.test.js

```diff
@@ -79,40 +71,25 @@ describe('Platform Routes HTTP request handler', () => {
       await assertResponseStatus(res, 200)
       const metrics = await res.json()
       assert.deepStrictEqual(metrics, [
-        { day: '2024-01-11', station_id_count: 1 },
-        { day: '2024-01-12', station_id_count: 2 }
+        { day: '2024-01-11', station_count: 3 },
+        { day: '2024-01-12', station_count: 4 }
       ])
     })
   })
 
   describe('GET /stations/monthly', () => {
-    it('returns monthly station metrics for the given date range ignoring the day number', async () => {
+    it('returns monthly station metrics for the given date range', async () => {
       // before the date range
-      await givenDailyStationMetrics(pgPools.evaluate, '2023-12-31', [
-        { ...STATION_STATS, acceptedMeasurementCount: 1, totalMeasurementCount: 1 }
-      ])
+      await givenMonthlyActiveStationCount(pgPools.evaluate, '2023-12-01', 1)
       // in the date range
-      await givenDailyStationMetrics(pgPools.evaluate, '2024-01-10', [
-        { ...STATION_STATS, acceptedMeasurementCount: 1, totalMeasurementCount: 1 }
-      ])
-      await givenDailyStationMetrics(pgPools.evaluate, '2024-01-11', [
-        { ...STATION_STATS, stationId: 'station2', acceptedMeasurementCount: 1, totalMeasurementCount: 1 }
-      ])
-      await givenDailyStationMetrics(pgPools.evaluate, '2024-01-12', [
-        { ...STATION_STATS, stationId: 'station2', acceptedMeasurementCount: 2, totalMeasurementCount: 2 },
-        { ...STATION_STATS, stationId: 'station3', acceptedMeasurementCount: 1, totalMeasurementCount: 1 }
-      ])
-      await givenDailyStationMetrics(pgPools.evaluate, '2024-02-13', [
-        { ...STATION_STATS, acceptedMeasurementCount: 1, totalMeasurementCount: 1 }
-      ])
+      await givenMonthlyActiveStationCount(pgPools.evaluate, '2024-01-01', 3)
+      await givenMonthlyActiveStationCount(pgPools.evaluate, '2024-02-01', 1)
       // after the date range
-      await givenDailyStationMetrics(pgPools.evaluate, '2024-03-01', [
-        { ...STATION_STATS, acceptedMeasurementCount: 1, totalMeasurementCount: 1 }
-      ])
+      await givenMonthlyActiveStationCount(pgPools.evaluate, '2024-03-01', 1)
```

Comment:
Let's use a value that's different from February.

```suggestion
      await givenMonthlyActiveStationCount(pgPools.evaluate, '2024-03-01', 4)
```